pkg/kubernetes/scripts/policy.js

```diff
@@ -174,6 +176,49 @@
                 return result;
             }
 
+            var sarCache = { };
+
+            function subjectAccessReview(project, user, verb, resource) {
```

Comment:
If we are going to have the first argument just be a string then I'd suggest calling it namespace. As project seems to indicate a project api object. Instead of just the namespace string.